antsibull/cli/doc_commands/devel.py

```diff
@@ -2,7 +2,140 @@
 # License: GPLv3+
 # Copyright: Ansible Project, 2020
 
+import asyncio
+import os
+import os.path
+import tempfile
+import typing as t
+
+import aiohttp
+import asyncio_pool
+
+from .stable import generate_docs_for_all_collections
+from ... import app_context
+from ...ansible_base import get_ansible_base
+from ...collections import install_together
+from ...compat import asyncio_run
+from ...dependency_files import parse_pieces_file
+from ...galaxy import CollectionDownloader, DownloadResults
+from ...logging import log
+from ...venv import VenvRunner
+
+if t.TYPE_CHECKING:
+    import semantic_version as semver
+
+
+mlog = log.fields(mod=__name__)
+
+
+async def retrieve(collections: t.List[str],
+                   tmp_dir: str,
+                   galaxy_server: str,
+                   ansible_base_source: t.Optional[str] = None,
+                   collection_cache: t.Optional[str] = None) -> t.Dict[str, 'semver.Version']:
+    """
+    Download ansible-base and the latest versions of the collections.
```

Comment:
What about ansible-core?

Reply:
I've changed it in ff2175ba6ed763766d45041fb29e8141bded1c7d in the places where the name can easily be adjusted. Replacing base with core is something that needs to be done in the remainder of antsibull as well.

Reply:
I agree with felixfontein's evaluation here.  Probably best to use one or two separate PRs to address the other places.